webpack/components/Content/ContentTable.js

```diff
@@ -20,7 +20,6 @@ const ContentTable = ({ content, tableSchema, onPaginationChange }) => {
       loadingText={__('Loading')}
     >
       <Table
-        ouiaId="content-table-table"
```

Comment:
Are these `ouiaId` part of `query-string`? I thought we needed them for QE to be happy?

Reply:
Normally we do need them, but this is a Patternfly 3 component and it causes a console error if I include it.